test/integration/scripts/int-test-setup.sh

```diff
@@ -175,6 +175,9 @@ echo ${TPM2TOOLS_TCTI}
 export TPM2_PKCS11_TCTI="tabrmd:${TABRMD_TEST_TCTI_CONF}"
 echo ${TPM2_PKCS11_TCTI}
 
+setup_fapi
+tss2_provision
```

Comment:
As im sure your already aware, but ill just mention it again, you'll also need tests at some point where the setup isn't already done.